### PR TITLE
(DOCSP-16408): Expose connection string for deployments for easy of use

### LINKED ISSUE #452: [questions] Seed lists?
Does this operator expose seed list DNS anywhere? Since that would make it easier to connect to an auto scaling replica set.

### PR DESCRIPTION
**Pull request by jwilliams-mongo**
[JIRA](https://jira.mongodb.org/browse/DOCSP-16408)

Closes #452 

### All Submissions:

* [ x] Have you opened an Issue before filing this PR?
* [x ] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ x] Have you signed all of your commits?
* [ x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ x ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: docs/deploy-configure.md [hunk 1]
```diff
@@ -24,12 +24,60 @@ To deploy your first replica set:
    ```
    kubectl get mongodbcommunity --namespace <my-namespace>
    ```
-4. Connect clients to the MongoDB replica set:
+
+4. The Community Kubernetes Operator creates secrets that contains user's connection strings and credentials.
+
+   The secrets follow this naming convention: `<metadata.name>-<auth-db>-<username>`, where:
```
Comment (MongoCaleb) [inline docs/deploy-configure.md:30]: Duplicate "where"s

File: docs/deploy-configure.md [hunk 2]
```diff
@@ -24,12 +24,60 @@ To deploy your first replica set:
...
+
+   | Variable | Description | Value in Sample |
+   |----|----|----|
+   | `<metadata.name>` | Name of the MongoDB database resource. | `example-mongodb` |
+   | `<auth-db>` | [Authentication database](https://docs.mongodb.com/manual/core/security-users/#std-label-user-authentication-database) where you defined the database user. | `admin` |
+   | `<username>` | Username of the database user. | `my-user` |
+
+   Update the variables in the following command, then run it to retrieve a user's connection strings to the replica set from the secret:
+
+   **NOTE**: The following command requires `[jq](https://stedolan.github.io/jq/)` version 1.6 or higher.</br></br>
```
Comment (MongoCaleb) [inline docs/deploy-configure.md:42]: This hyperlink is not rendering properly.  Perhaps because it is in a code block?

**Review by MongoCaleb — APPROVED** [review]
2 things that need fixing, but otherwise LGTM

File: docs/deploy-configure.md [hunk 3]
```diff
@@ -24,12 +24,58 @@ To deploy your first replica set:
    ```
    kubectl get mongodbcommunity --namespace <my-namespace>
    ```
-4. Connect clients to the MongoDB replica set:
+
+4. The Community Kubernetes Operator creates secrets that contains user's connection strings and credentials.
```
Comment (chatton) [inline docs/deploy-configure.md:28]: there can be multiple users so I believe this should be `that contain users' connection strings` (I could be mistaken!)

**Review by chatton — APPROVED** [review]
LGTM! I just had one grammar note - I am not an expert so could be mistaken, if I will leave it up to you to correct if necessary and merge, thanks for the work on this! 👍

**Review by tibulca — APPROVED** [review]

File: docs/deploy-configure.md (hunk 3 above)
Comment (jwilliams-mongo) [reply docs/deploy-configure.md]: @chatton I had it this way at first. Your comment affirms that I should go with that. updated.

**Pull request merged**